api/src/main/java/facade/CarFacade.java

```diff
@@ -0,0 +1,52 @@
+package facade;
+
+import cz.muni.fi.entities.Driver;
+import dto.CarDTO;
+
+import java.util.List;
+
+/**
+ * @author Richard Hrmo
+ */
+public interface CarFacade {
```

Comment:
We're supposed to implement even more complex methods not just basics. Please try to come up with something more advanced.